frontend/src/data/utils/datasetutils.ts

```diff
@@ -201,6 +201,13 @@ const NON_STANDARD_AND_MULTI: DemographicGroup[] = [
   MULTI_OR_OTHER_STANDARD_NH,
 ]
 
+const VARIABLES_NEEDING_13PLUS: VariableId[] = [
```

Comment:
Can we avoid duplication between this array and the one in the HIVSuffix hook? https://github.com/SatcherInstitute/health-equity-tracker/blob/b4cee7f337b2d2dbe8e915ffe77dd8db6629ebee/frontend/src/utils/hooks/useHIVLabelSuffix.tsx#L8

I feel like we should define global constants that are topic-specific in their Frontend provider file; and then export/import them as needed throughout the app